packages/core/src/types.ts

```diff
@@ -134,60 +135,7 @@ export interface UserOperationReceipt {
   /* The logs generated by this UserOperation (not including logs of other UserOperations in the same bundle). */
   logs: string[];
   /* The TransactionReceipt object for the entire bundle, not only for this UserOperation. */
-  receipt: UserOperationReceiptObject;
-}
-
-export interface UserOperationReceiptObject {
```

Comment:
this is same as viem's TransactionReceipt

Reply:
The issue with deleting these types is if anyone was importing them before they now would end up with a breaking change

Reply:
got it. will put it back but put deprecated tag